src/Distributions.jl

```diff
@@ -7,6 +7,7 @@ using StatsFuns
 using StatsBase
 using Compat
 
+import Compat.view 
 import Base.Random
 import Base: size, eltype, length, full, convert, show, getindex, scale, scale!, rand, rand!
 if VERSION > v"0.4"
```

Comment:
If it's v0.4, it's going to pick from compat. Nothing happens in v0.5. So this would work.